src/components/menu/examples/menu-size.tsx

```diff
@@ -0,0 +1,57 @@
+import { MenuItem, ListSeparator } from '@limetech/lime-elements';
+import { Component, Element, Host, State, h } from '@stencil/core';
+
+/**
+ * Resize the menu drop-down
```

Comment:
I suggest calling this example "Set custom width of drop-down" or something along those lines. "Resize" makes me think the user can resize it or something.

Reply:
The commit message could also be improved I think. From `feat(menu): support custom width of content` to something like `feat(menu): support setting custom width of drop-down`, perhaps? 🤷‍♂️

Reply:
fixed